src/plugins/simulator/visualizations/qt-opengl/qtopengl_camera.h

```diff
@@ -37,85 +37,54 @@ namespace argos {
       struct SPlacement {
          /** The position of the camera in the global reference frame */
          CVector3 Position;
-         /** The local Y axis of the camera in the global reference frame */
-         CVector3 Left;
+         /** What we are looking at in the global reference frame */
+         CVector3 Target;
          /** The local Z axis of the camera in the global reference frame */
          CVector3 Up;
-         /** The local X axis of the camera in the global reference frame */
-         CVector3 Forward;
-         /** The direction of sight of the camera in the global reference frame */
-         CVector3 Target;
          /** The focal length of the lens (if this was a real camera) */
          Real LensFocalLength;
          /** The focal length of the camera */
          CDegrees YFieldOfView;
-         /** Motion sensitivity */
```

Comment:
Why did you remove this?

Reply:
It is only needed during `move`/`rotate` so it is now calculated on demand. It doesn't make sense to recalculate it every time active camera placement is updated during interpolate.